tools/StaticAnalysis/Exceptions/BreakingChangeIssues.csv

```diff
@@ -18,4 +18,12 @@
 "Microsoft.Azure.Commands.Management.CognitiveServices.dll","Microsoft.Azure.Commands.Management.CognitiveServices.SetAzureCognitiveServicesAccountCommand","Set-AzureRmCognitiveServicesAccount","0","3000","The type of property 'AccountType' of type 'PSCognitiveServicesAccount' has changed from 'System.Nullable`1[Microsoft.Azure.Management.CognitiveServices.Models.Kind]' to 'System.String'.","Change the type of property 'AccountType' back to 'System.Nullable`1[Microsoft.Azure.Management.CognitiveServices.Models.Kind]'."
 "Microsoft.Azure.Commands.Compute.dll","Microsoft.Azure.Commands.Compute.Extension.Chef.SetAzureVMChefExtensionCommand","Set-AzureRmVMChefExtension","0","2050","A validate set has been added for parameter 'Daemon' for cmdlet 'Set-AzureRmVMChefExtension'.","Remove the validate set from parameter 'Daemon'."
 "Microsoft.Azure.Commands.Compute.dll","Microsoft.Azure.Commands.Compute.Extension.Chef.SetAzureVMChefExtensionCommand","Set-AzureRmVMChefExtension","0","2090","The ValidateNotNullOrEmpty attribute has been added to parameter 'Linux' for cmdlet 'Set-AzureRmVMChefExtension'.","Remove the ValidateNotNullOrEmpty attribute from parameter 'Linux'."
-"Microsoft.Azure.Commands.Compute.dll","Microsoft.Azure.Commands.Compute.Extension.Chef.SetAzureVMChefExtensionCommand","Set-AzureRmVMChefExtension","0","2090","The ValidateNotNullOrEmpty attribute has been added to parameter 'Windows' for cmdlet 'Set-AzureRmVMChefExtension'.","Remove the ValidateNotNullOrEmpty attribute from parameter 'Windows'."
\ No newline at end of file
+"Microsoft.Azure.Commands.Compute.dll","Microsoft.Azure.Commands.Compute.Extension.Chef.SetAzureVMChefExtensionCommand","Set-AzureRmVMChefExtension","0","2090","The ValidateNotNullOrEmpty attribute has been added to parameter 'Windows' for cmdlet 'Set-AzureRmVMChefExtension'.","Remove the ValidateNotNullOrEmpty attribute from parameter 'Windows'."
+"Microsoft.Azure.Commands.Network.dll","Microsoft.Azure.Commands.Network.GetAzureExpressRouteCircuitARPTableCommand","Get-AzureRmExpressRouteCircuitARPTable","0","2030","The parameter 'Name' is no longer optional for the parameter set '__AllParameterSets' for cmdlet 'Get-AzureRmExpressRouteCircuitARPTable'.","Make 'Name' optional for the parameter set '__AllParameterSets'."
+"Microsoft.Azure.Commands.Network.dll","Microsoft.Azure.Commands.Network.GetAzureExpressRouteCircuitARPTableCommand","Get-AzureRmExpressRouteCircuitARPTable","0","2080","The parameter 'Name' in parameter set '__AllParameterSets' for cmdlet 'Get-AzureRmExpressRouteCircuitARPTable' no longer has the attribute 'ValueFromPipelineByPropertyName'.","Add the attribute 'ValueFromPipelineByPropertyName' back to parameter 'Name' in parameter set '__AllParameterSets'."
```

Comment:
These exceptions around GetValueByPipelineByPropertyName still apply - ExpressRouteCircuitName shoudl allow getting its value from the pipeline by property name -> this allows piping in a Get from generic cmdlets into these methods.

The other exceptions are valid